src/coins/coins.ts

```diff
@@ -39,7 +39,7 @@ export const basicCoins: BasicCoin[] = [
       [ChainId.SOL]: {
         address: '7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs',
         decimals: 8,
-        name: 'Wrapped SOL (Wormhole)',
+        name: 'Ether (Wormhole)',
```

Comment:
Are you sure this is Ether? ChainId says it is SOL 😅

Reply:
Isn't this supposed to be ETH on Solana 🤔 https://solana.fm/address/7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs?cluster=mainnet-solanafmbeta

Reply:
Good question, maybe we should change ChainId as well then?